data_summaries/Lumos Planning.json

```diff
@@ -0,0 +1,168 @@
+{
+    "lumos-planning-complex_qa": {
+        "Unique Dataset Identifier": "lumos-planning-complex_qa",
+        "Dataset Name": "ai2lumos/lumos_unified_plan_iterative",
+        "Paper Title": "Lumos: Learning Agents with Unified Data, Modular Design, and Open-Source LLMs",
+        "Dataset URL": "https://huggingface.co/datasets/ai2lumos/lumos_unified_plan_iterative",
+        "GitHub URL": "https://github.com/allenai/lumos",
+        "Hugging Face URL": "https://huggingface.co/datasets/ai2lumos/lumos_unified_plan_iterative",
+        "Papers with Code URL": "",
+        "ArXiv URL": "https://arxiv.org/abs/2311.05657",
+        "Semantic Scholar Corpus ID": 265128672,
+        "Collection": "Lumos Planning",
+        "Collection URL": "https://huggingface.co/datasets/ai2lumos/lumos_unified_plan_iterative",
+        "Languages": [
+            "English"
+        ],
+        "Task Categories": [
+            "Logical Reasoning Question Answering",
+            "Binary Question Answering",
+            "Open-Domain Question Answering",
+            "Dialogue Generation",
+            "Instruction Following"
+        ],
+        "Text Sources": [
+            "ML Datasets"
+        ],
+        "Model Generated": [
+            "OpenAI GPT-4"
+        ],
+        "Format": [
+            "Multi-turn Dialog"
+        ],
+        "Human Annotation": "No",
+        "Derived from Datasets": [
+            "StrategyQA",
+            "Musique"
+        ],
+        "Creators": [
+            "AI2",
+            "UCLA",
+            "University of Washington"
+        ],
+        "Licenses": [
+            {
+                "License": "Apache License 2.0",
+                "License URL": "https://huggingface.co/datasets/ai2lumos/lumos_unified_plan_iterative"
+            },
+            {
+                "License": "OpenAI",
+                "License URL": ""
+            }
+        ],
```

Comment:
We should add the licenses for Mystique and StrategyQA, e.g. https://github.com/Data-Provenance-Initiative/Data-Provenance-Collection/blob/main/data_summaries/Flan%20Collection%20(Chain-of-Thought).json#L127